x/concentrated-liquidity/tick_test.go

```diff
@@ -1171,7 +1171,7 @@ func (s *KeeperTestSuite) TestGetTickLiquidityNetInDirection() {
 			// Normally, initialized during position creation.
 			// We only initialize ticks in this test for simplicity.
 			curPrice := sdk.OneDec()
-			curTick, err := math.PriceToTick(curPrice, pool.GetTickSpacing())
+			curTick, err := math.PriceToTickRoundUp(curPrice, pool.GetTickSpacing())
```

Comment:
Would be great to include an in line comment here as well for why we chose round up

Reply:
Added, there is no reason since tick spacing is always 1 so converted to regular `PriceToTick`. Added TODO to add tests with tick spacing > 1